File: MunchkinLib/MunchkinLib/Entities/Card.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MunchkinLib.Entities
{
	/**
	 @class Содержит описание карт
	 */
	abstract class Card
	{
		public Player Owner { get; set; }
		public String Name { get; private set; }
		public String Description { get; private set; }
	}
}

File: MunchkinLib/Cards/card.cpp
#include "card.h"


Card::Card()
{
    _id = 0;
    _name = "";
    _type = false;
}

Card::Card(uint id, QString name, bool type)
{
    _id = id;
    _name = name;
    _type = type;
}

QString Card::name() const
{
    return _name;
}

uint Card::id() const
{
    return _id;
}

bool Card::type() const
{
    return _type;
}
